src/FSharp.Formatting.ApiDocs/GenerateModel.fs

```diff
@@ -1019,10 +1048,16 @@ type internal CrossReferenceResolver(root, collectionName, qualify, extensions)
                 | true, (:? FSharpEntity as entity) ->
                     let urlBaseName = getUrlBaseNameForRegisteredEntity entity
 
-                    Some
+                    // See if we find the member that was intended, otherwise default to containing entity
+                    tryGetShortMemberNameFromMemberName memberName
+                    |> Option.bind (fun shortName ->
+                        findIList (entity.MembersFunctionsAndValues) (fun mfv -> mfv.DisplayName = shortName))
+                    |> Option.map mfvToCref
```

Comment:
A bit of personal a preference but I would go with:
```fsharp
|> function
    | Some mb -> Some(mfvToCref mb)
    | None ->
        Some
            { IsInternal = true
              ReferenceLink = internalCrossReference urlBaseName
              NiceName = getMemberName 2 entity.HasFSharpModuleSuffix memberName }
```

Just to make it explicit that you are handling both cases.
The code still compiles if you removed the `|> Option.defaultValue`, not handling the `None` case in the `function` will yield a warning/error.